programmers/2일차/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,16 @@
+function solution(s) {
+    let answer = "";
+    let splitS = s.split("");
+    if (splitS.length % 2 !== 0 ) {
+            answer = splitS[parseInt(splitS.length / 2)];
```

Comment:
```suggestion
            return splitS[parseInt(splitS.length / 2)];
```
answer 변수에 담을 필요 없이 바로 이렇게 return 해주셔도 됩니다